openassessment/workflow/models.py

```diff
@@ -49,12 +50,14 @@
 # We then use that score as the student's overall score.
 # This Django setting is a list of assessment steps (defined in `settings.ORA2_ASSESSMENTS`)
 # in descending priority order.
-DEFAULT_ASSESSMENT_SCORE_PRIORITY = ['peer', 'self', 'ai']
+DEFAULT_ASSESSMENT_SCORE_PRIORITY = ['staff', 'peer', 'self', 'ai']
 ASSESSMENT_SCORE_PRIORITY = getattr(
```

Comment:
Maybe we should just drop this setting since it seems to open up a lot of potential bugs... it isn't clear to me that all of our code really obeys this setting, and there aren't any tests for what happens when it is set.